sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp

```diff
@@ -110,30 +110,82 @@ namespace Azure { namespace Storage { namespace Test {
 
   TEST_F(BlockBlobClientTest, DownloadTransactionalHash)
```

Comment:
could we add some tests for the non 200 case to make sure we have the desired behavior ?

Reply:
done